client/my-sites/marketing/main.jsx

```diff
@@ -99,19 +101,22 @@ export const Sharing = ( {
 					</NavTabs>
 				</SectionNav>
 			) }
-			<UpgradeNudge
-				event="sharing_no_ads"
-				feature={ FEATURE_NO_ADS }
-				message={ translate( 'Prevent ads from showing on your site.' ) }
-				title={ translate( 'No Ads with WordPress.com Premium' ) }
-			/>
+			{ ! isVip && (
```

Comment:
I'm not sure but would it be better to add this logic to affect all Upgrade Nudges instead?

https://github.com/Automattic/wp-calypso/blob/94b2db408eed48de4fc5b821f10f9fc1e1b20093/client/blocks/upgrade-nudge/index.jsx#L92-L98

Reply:
We talked about this and we decided to update each parent component instead.

Reply:
@enejb can you expand a little bit the reasoning behind that decision? I think the maintenance cost will be lower if we delegate the VIP check to the upgrade banner itself. For example, right now there are bunch of banners that are still displayed because they have not been covered on this PR: single theme screen (`/theme/:theme/:site`), plugin upload (`/plugins/upload/:site`), theme upload (`/themes/upload/:site`).

Reply:
The main reasoning for me was that we would be able to control the notification. I didn't realize that we have logic in the `upgrade nudge` component that deals with displaying the nudge. I think it definitely makes sense to have the logic for VIP sites as well since I don't think we should ever be showing the notice to VIP site users at any point.